### PR TITLE
Release 1.4.13

### PR DESCRIPTION
**Pull request by mcmcphillips**
### IF THIS IS A RELASE, PR MUST BE NAMED `Release x.y.z`, where x.y.z is the new version

Jira Ticket: [MAT-7492](https://jira.cms.gov/browse/MAT-7492)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Review by sb-cecilialiu — APPROVED** [review]

File: src/BaseConfigurationTypes.ts [hunk 1]
```diff
@@ -2,10 +2,10 @@ export enum BaseConfigurationTypes {
   APPROPRIATE_USE_PROCESS = "Appropriate Use Process",
   RESOURCE_USE = "Resource Use",
   EFFICIENCY = "Efficiency",
-  INTERMEDIATE_CLINICAL_OUTCOME = "Intermediate Clinical Outcome",
+  INTERMEDIATE_CLINICAL_OUTCOME = "Intermediate Outcome",
   OUTCOME = "Outcome",
   EXPERIENCE = "Experience",
-  PATIENT_REPORTED_OUTCOME_PERFORMANCE = "Patient Reported Outcome Performance",
+  PATIENT_REPORTED_OUTCOME_PERFORMANCE = "Patient Reported Outcome",
```
Comment (adongare) [inline src/BaseConfigurationTypes.ts:8]:
```suggestion
  PATIENT_REPORTED_OUTCOME = "Patient Reported Outcome",
```

File: src/BaseConfigurationTypes.ts [hunk 2]
```diff
@@ -2,10 +2,10 @@ export enum BaseConfigurationTypes {
   APPROPRIATE_USE_PROCESS = "Appropriate Use Process",
   RESOURCE_USE = "Resource Use",
   EFFICIENCY = "Efficiency",
-  INTERMEDIATE_CLINICAL_OUTCOME = "Intermediate Clinical Outcome",
+  INTERMEDIATE_CLINICAL_OUTCOME = "Intermediate Outcome",
```
Comment (adongare) [inline src/BaseConfigurationTypes.ts:5]:
```suggestion
  INTERMEDIATE_OUTCOME = "Intermediate Outcome",
```

**Review by adongare — APPROVED** [review]
